webpack/assets/javascripts/react_app/components/common/forms/Select.js

```diff
@@ -17,7 +17,7 @@ class Select extends React.Component {
     const renderOptions = arr =>
       map(arr, (attribute, value) => (
         <option key={attribute} value={value}>
-          {value}
+          {attribute}
```

Comment:
Will it always be? in other words - is the issue here in the implementation of select taking the wrong attribute or of vmware passing the wrong attribute?

Reply:
You are right. It would always take the attribute for showing the text. I think this is OK. Somehow it must be possible to set a different text than using the boring value.